docs/releasenotes/index.rst

```diff
@@ -10,11 +10,15 @@ You can learn more about where we might be going with future releases by:
 * Stopping by the ``#clusterhq`` channel on ``irc.freenode.net``.
 * Visiting our GitHub repository at https://github.com/ClusterHQ/flocker.
 
-Next release
-============
+v1.4.0
+======
 
 * The :ref:`dataset API <api>` added support for leases.
   Leases prevent a dataset from being deleted or moved off a node.
+* Fix line splitting when logging to systemd's journal.
```

Comment:
"systemd" triggers the spellcheck. Add double backquotes around it.  You may need to rephrase it slightly to avoid possessive with backquoted word.